src/nvim/buffer.c

```diff
@@ -5361,8 +5372,8 @@ bool bt_terminal(const buf_T *const buf)
   return buf != NULL && buf->b_p_bt[0] == 't';
 }
 
-// Return true if "buf" is a "nofile", "acwrite" or "terminal" buffer.
-// This means the buffer name is not a file name.
+// Return TRUE if "buf" is a "nofile", "acwrite", "terminal" or "prompt"
```

Comment:
`bt_nofile()` is a boolean function so it returns true/false, not TRUE/FALSE. Vim uses TRUE/FALSE to support compilers that don't support `bool`. Neovim targets C99 so `bool` is supported.